### DISCUSSION TITLE
zoom link

### DISCUSSION THREAD

**Discussion by anlhu** [casa-business-manager/CASA#59] [General]
https://ucsc.zoom.us/j/2740605731?pwd=YlFNZjdXbEdKeVJYY1RSc2F2Mm9Xdz09

**Comment by catdiehl** [comment]
in case you didnt see ian said he is good to meet tuesday evening malaysia time

**Reply by anlhu** [reply]
Oh crap so about 10 hours from now?

**Reply by catdiehl** [reply]
no 8am US PST wednesday the original time we wanted

**Reply by catdiehl** [reply]
i dont know why i wrote tuesday i think i confused myself but the time we wanted originally

**Comment by catdiehl** [comment]
[Scrum Board.pdf](https://github.com/user-attachments/files/16530744/Scrum.Board.pdf)

**Reply by catdiehl** [reply]
<img width="607" alt="image" src="https://github.com/user-attachments/assets/a2ab5a05-8efc-4bec-832a-aafc8d2cca78">
